content/en/Getting started/Assets/coverage.md

```diff
@@ -75,10 +75,6 @@ different asset sizes and coverage levels:
 {{% pentest-report-requirements %}}
 <!-- For content, see the following file: layouts/shortcodes/pentest-report-requirements.html -->
 
-If you want a pentest report, you must set up a test of at least two credits. 
```

Comment:
I've removed this paragraph, as it's a duplicate of the content pulled using {{% pentest-report-requirements %}}

To see the resulting duplication for yourself, see https://developer.cobalt.io/getting-started/assets/coverage/#pentest-reports.